.github/workflows/export_data.yml

```diff
@@ -30,15 +30,20 @@ jobs:
           python -m pip install --upgrade wheel
           python -m pip install .
 
+      - name: Try single CLI run of tool
+        run: |
+          NO_EXIT_CVE_NUM=1 python -m cve_bin_tool.cli -u now test/assets/test-kerberos-5-1.15.1.out
```

Comment:
How do you feel about integrating this job with the cache job?  Two possible ways:
1. actually  using the cache here as a starting seed so we don't have to get everything
2. getting rid of export_data.xml and integrating this code into the existing cache job

I think maybe number 2 will be the right way to go when we get this automated and running daily.

Reply:
@terriko the problem with that would be you need to merge a pull request every single day and if you miss 2-3 days in a row there will be a mess